specificatie/openapi.yaml

```diff
@@ -694,6 +707,94 @@ components:
           type: string
           pattern: '^[0-9]{16}$'
         example: ['0226100000008856']
+      explode: false  
+
+    geconstateerd:
+      description: 'Geeft aan of naar geconstateerde objecten moet worden gezocht.'
+      name: geconstateerd
+      in: query
+      required: false
+      schema:
+        type: boolean
+        
+    statusPand:
+      description: "Status van het pand."
+      name: statusPand
+      in: query
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items:
+          $ref: '#/components/schemas/StatusPandEnum'
+
+    bouwjaar:
```

Comment:
Is er een min en max waarde voor bouwjaar? Ik neem aan dat bouwjaar[min]=0 en bouwjaar[max]=99999 geen geldig bouwjaren zijn

Reply:
Het formele type is een Year, volgens ISO 8601 mag een year een waarde hebben tussen 0 en 9999. In de praktijk komt bouwjaar 0 natuurlijk niet voor, maar mag hier wel als ondergrens worden gegeven. Ik heb de minimale en maximale waarden erbij vermeld.